### DISCUSSION TITLE
Issues with release not working

### DISCUSSION THREAD

**Discussion by marksie1988** [semantic-release/semantic-release#2942] [Q&A] [answered]
Hi All, 

I have created a github action which was working no problem, however yesterday i noticed that the release never happened and the action stopped here:

```
[4:49:37 PM] [semantic-release] › ℹ  Found 10 commits since last release
[4:49:37 PM] [semantic-release] › ℹ  Start step "analyzeCommits" of plugin "@semantic-release/commit-analyzer"
```

I enabled debug mode on below run: 
https://github.com/totaldebug/atomic-calendar-revive/actions/runs/6087527153/job/16516276180

When i run it locally the same thing happens, it just stops. 

I am new to semantic-release so not sure if there is a config issue or a bug because like i say it was working but stopped yesterday. 

any help would be really appreciated

**Comment by travi** [comment, marked as answer]
Please see #2932 and the likely related #2929

**Reply by marksie1988** [reply]
Ah ok I'll give that workaround a try and see if it fixes it. Thanks

**Reply by marksie1988** [reply]
Using the beta version has resolved the issue thanks.

**Reply by travi** [reply]
Thanks for confirming
